index.js

```diff
@@ -43,7 +45,7 @@ module.exports = function parseDate (isoDate) {
     date = new Date(year, month, day, hour, minute, second, ms)
   }
 
-  if (isFirstCentury) {
+  if (yearIsBetween0And99) {
```

Comment:
Would rather remove this var and just move the check here, since it's not reused